client/dashboard/app/style.scss

```diff
@@ -18,6 +18,25 @@
 	text-transform: uppercase;
 }
 
+// Core CheckboxControl's :disabled styles are overridden by its :checked styles. This is our workaround.
```

Comment:
Ah... this is now broken when the screen is accessed from v1 hosting dashboard 😄 

<img width="704" height="517" alt="Image" src="https://github.com/user-attachments/assets/91682ad1-5f98-4519-8af0-f3d5006351af" />

I *think* we also should duplicate this here as well: https://github.com/Automattic/wp-calypso/blob/914817d2ff9d32b8f4a286ccd90f7fc1d578a6e8/client/sites/v2/site-settings/style.scss